eng/Versions.props

```diff
@@ -31,11 +31,11 @@
     <MicrosoftCodeAnalysisNetAnalyzersVersion>6.0.0-preview1.21054.10</MicrosoftCodeAnalysisNetAnalyzersVersion>
     <MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.20623.3</MicrosoftCodeAnalysisTestingVersion>
     <CodeStyleAnalyzerVersion>3.8.0</CodeStyleAnalyzerVersion>
-    <VisualStudioEditorPackagesVersion>16.9.220</VisualStudioEditorPackagesVersion>
+    <VisualStudioEditorPackagesVersion>16.10.44</VisualStudioEditorPackagesVersion>
```

Comment:
Does this need to merge into main-vs-deps?

Reply:
Is there any risk to merge this to main-vs-deps?

Reply:
roslyn needs to be runnable on 16.9.  So if these introduce 16.10 dependencies that are not present in 16.9 and fail to load, then this needs to go into the -vs-deps branch.  I presume since these are editor dependencies 16.9 will fall on the floor